client/my-sites/upgrades/checkout-thank-you/domain-registration-details.jsx

```diff
@@ -7,34 +7,72 @@ import React from 'react';
  * Internal dependencies
  */
 import { getDomainManagementUrl } from './utils';
-import { isPlan } from 'lib/products-values';
+import { isGoogleApps } from 'lib/products-values';
 import i18n from 'lib/mixins/i18n';
 import PurchaseDetail from 'components/purchase-detail';
+import userFactory from 'lib/user';
+
+const user = userFactory();
+
+const DomainRegistrationDetails = ( { selectedSite, domain, purchases } ) => {
+	const googleAppsWasPurchased = purchases.some( isGoogleApps ),
+		hasOtherPrimaryDomain = selectedSite.options.is_mapped_domain && selectedSite.domain !== domain;
 
-const DomainRegistrationDetails = ( { selectedSite, domain } ) => {
 	return (
 		<div>
 			<PurchaseDetail
+				icon="notice"
+				title={ i18n.translate( 'Verify your email address' ) }
+				description={ i18n.translate( 'We sent you an email with a request to verify your new domain. Unverified domains may be suspended.' ) }
+				buttonText={ i18n.translate( 'Learn more about verifying your domain' ) }
+				href="//support.wordpress.com/domains/register-domain/#email-validation-and-verification"
+				target="_blank"
+				requiredText={ i18n.translate( 'Important! Your action is required.' ) } />
+
+			{ googleAppsWasPurchased && (
+				<PurchaseDetail
+					icon="cog"
+					title={ i18n.translate( 'Finish setting up your Google Apps account' ) }
+					description={
+						i18n.translate(
+							'We emailed you at %(email)s with login information for your new Google Apps account. ' +
+							'If you can\'t find it, try a global search for "Google Apps".',
+							{
+								args: { email: user.get().email }
+							}
+						)
+					}
+					buttonText={ i18n.translate( 'Learn more about Google Apps' ) }
+					href="//apps.google.com/learning-center/"
+					target="_blank"
+					requiredText={ i18n.translate( 'Important! Your action is required.' ) } />
```

Comment:
Good catch. What do you think about creating a new sub-component that is a child of both `GoogleAppsDetails` and `DomainRegistrationDetails`? This would get around another issue:

- Importing a `Details`-suffix component in another `Details`-suffix component creates a dependency between components which are siblings in both directory structure and use the same naming convention, which makes it non-obvious which of these components do/can import each other.
- We need to pass different props to this `PurchaseDetail`, depending on where it is mounted.